scripts/script.yaml

```diff
@@ -99,76 +99,101 @@
+                                                placeholder: "מספר החולה שפגשתי"
+                                            - say: האם ידוע לך התאריך שבו נפגשתם?
+                                            - wait:
+                                                options:
+                                                - show: "כן"
+                                                  steps: 
+                                                  - say: באיזה תאריך?
+                                                  - wait:
+                                                      variable: insulation_exposure_date
+                                                      placeholder: "תאריך: dd.mm.yy"
+                                                - show: "לא"
+                                        - show: "לא"
+                        - show: אני נמצא בבידוד כי חוויתי תסמינים
+                          value: has_symptoms
+                        - show: אני נמצא בבידוד מרצוני האישי                    
+                          value: voluntary
+                - show: "אני חולה קורונה (מאומת בבדיקת מעבדה)"
+                  value: diagnosed
                   steps:
-                    - wait:
-                        variable: hospitalized
-                        options:                    
-                        - show: אובחנתי כחולה בבדיקת מעבדה לקורונה – באישפוז בבית חולים
-                          value: hospital_from_corona_lab
-                        - show: אובחנתי כחולה –  באישפוז בבית חולים
-                          value: hospital
-                        - show: אובחנתי כחולה - באישפוז במלון
-                          value: hotel
-                        - show: אובחנתי כחולה - בבידוד בבית
-                          value: home
+                    - say: והיכן אתם נמצאים?
+                      wait:
+                        variable: diagnosed_location
+                        option:                 
+                        - show: אשפוז בבית חולים
+                          value: diagnosed_hospital
+                        - show: בבית מלון
+                          value: diagnosed_hotel
+                        - show: בבית
+                          value: diagnosed_home
+                        - show: החלמתי ואני בבית
+                          value: diagnosed_recovered
+                - show: "אני לא בבידוד"
+                  value: "None"
 
       - name: current-report
         steps:
           - say: מה שלומך היום?
           - wait:
+              variable: general_feeling
               options:
-                - show: מצויין!
-                - show: לא כל כך טוב
-                  steps:                    
-                    - goto: current-report-symptoms
+              - show: מצויין!
+                variable: feel_good
+                step:
+                - say: "שמחים לשמוע. בכל זאת נרצה לברר אם יש לך משהו מהתסמינים הללו:"
+                - goto: current-report-symptoms
```

Comment:
no need for goto on both branches

Reply:
@akariv missed this comment before, sorry.

turned into a `switch`